### PR TITLE
Set up docker image publishing

### PR DESCRIPTION
**Pull request by Ge11ert**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM node:14.18.0-stretch-slim
+
+ARG CRAFTER_VERSION=latest
+ARG BUILD_DATE
+
+LABEL org.opencontainers.image.title="Crafter" \
+      org.opencontainers.image.description="An image to launch \"@funboxteam/crafter\" npm package" \
+      org.opencontainers.image.vendor="FunBox" \
+      org.opencontainers.image.licenses="MIT" \
```
Comment (Ge11ert) [inline docker/Dockerfile:9]: At first, we had been using MIT license for our open-source projects, but later we switched to Apache-2.0 (see https://github.com/funbox/blueprinter).

Should we switch to Apache-2.0 here too?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@funboxteam/crafter",
-  "version": "4.1.1",
+  "version": "4.1.2",
```
Comment (Ge11ert) [inline package.json:3]: We need to bump package version in order to obtain successful "npm publish" pipeline which is a trigger to build and push docker image.

File: docker/Dockerfile (hunk 1 above)
Comment (igoradamenko) [reply docker/Dockerfile:9]: We'd prefer to use Apache-2.0 for our opensource projects. If it's possible to switch here, let's do it.

**Review by igoradamenko — APPROVED** [review]

File: docker/Dockerfile (hunk 1 above)
Comment (igoradamenko) [reply docker/Dockerfile:9]: However, let's switch the license using a separate PR. 

It feels like a major change.

File: .github/workflows/push-docker.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Workflow is triggered after a new Crafter version is successfully published to NPM
+
+name: Build and publish Docker image
+
+on:
+  workflow_run:
+    workflows: [Publish Package to npmjs]
```
Comment (igoradamenko) [inline .github/workflows/push-docker.yml:7]: Ah, missed the wrong title here.

Comment (igoradamenko) [reply .github/workflows/push-docker.yml:7]: Oh, wait, I'm dumb, it's a dependency.

Never mind.

**Comment by igoradamenko** [comment]
image

[It works.](https://hub.docker.com/r/funbox/crafter/tags)

File: docker/Dockerfile (hunk 1 above)
Comment (Ge11ert) [reply docker/Dockerfile:9]: https://github.com/funbox/crafter/pull/14

**Pull request merged**